### Get backend chess engine's move service.

---
ChessMoveMsg move   # engine's fully quantified move
